spec/factories/posts.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :post do
+    title { Faker::Hacker.abbreviation }
+    body { Faker::Hacker.say_something_smart }
+    avatar { Faker::Avatar.image }
+    category nil
```

Comment:
You don't need to specify `category nil`. You will get same result if you remove this line

Reply:
OK, removed.